BParkingNano/python/trgbits_cff.py

```diff
@@ -0,0 +1,17 @@
+import FWCore.ParameterSet.Config as cms
+from PhysicsTools.NanoAOD.common_cff import *
+
+
+
+trgTable = cms.EDProducer( "TrgBitTableProducer",
+                          triggerresults= cms.InputTag("TriggerResults::HLT"),
+                           paths= cms.vstring("HLT_Mu9_IP6",    "HLT_Mu9_IP5", 
+                                              "HLT_Mu7_IP4",    "HLT_Mu8_IP3",
+                                              "HLT_Mu12_IP6"
+                                              ),
```

Comment:
The list of paths that I had some larger...   maybe @sarafiorendi can cross-check
HLT_Mu7_IP4
HLT_Mu8_IP3     HLT_Mu8_IP6     HLT_Mu8_IP5     HLT_Mu8p5_IP3p5   
HLT_Mu9_IP6     HLT_Mu9_IP4    HLT_Mu9_IP5
HLT_Mu10p5_IP3p5
HLT_Mu12_IP6

Reply:
I confirm the list of path from Arabella is the complete one